src/moonstone-samples/lib/CalendarSample.js

```diff
@@ -118,7 +118,7 @@ module.exports = kind({
 	setLocale: function(inSender, inEvent){
 		if (ilib) {
 			var locale = inEvent.selected.content,
-				val = (locale == 'Use Default Locale') ? null : locale;
+				val = (locale == 'Use Default Locale') ? '' : locale;
```

Comment:
@ehoogerbeets @gouniLee: When calling `ilib.setLocale` and passing a falsy value such as `null`, was there a change such that only the empty string `''` would cause the current locale to be used, or was this always the case? My memory is fuzzy, but I thought we had made a change at one point where passing in a falsy value would result in the use of the current locale.